src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomBtfBlock.java

```diff
@@ -0,0 +1,62 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+
+import org.testng.annotations.Test;
+
+public class TestAdsFandomBtfBlock extends AdsFandomTestTemplate {
+
+  private void getJquery() {
+    driver.executeScript(
+        "    (function () {\n"
+        + "    var s = document.createElement('script');\n"
+        + "    s.type = 'text/javascript';\n"
+        + "    s.async = true;\n"
+        + "    s.src = 'https://code.jquery.com/jquery-2.2.4.min.js';\n"
+        + "    var x = document.getElementsByTagName('script')[0];\n"
+        + "    x.parentNode.insertBefore(s, x);\n"
+        + "    })();"
+    );
+  }
+
+  @Test(
+      dataProviderClass = FandomAdsDataProvider.class,
+      dataProvider = "fandomBtfBlockPage",
+      groups = {"AdsFandomBtfBlockDesktop"}
+  )
+  public void adsFandomBtfBlockDesktop(String article) {
+    String testedPage = urlBuilder.getUrlForFandomPage(article);
+    AdsFandomObject fandomPage = new AdsFandomObject(driver, testedPage);
+    getJquery();
+
+    fandomPage.verifyTopLeaderboard();
+    fandomPage.verifyTopBoxad();
+    fandomPage.verifyNoIncontentBoxad();
+    fandomPage.verifyNoBottomLeaderboard();
+  }
+
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  @Test(
+      dataProviderClass = FandomAdsDataProvider.class,
+      dataProvider = "fandomBtfBlockPage",
+      groups = {"AdsFandomBtfBlockMobile"}
+  )
+  public void adsFandomBtfBlockMobile(String article) {
+    String testedPage = urlBuilder.getUrlForFandomPage(article);
+    AdsFandomObject fandomPage = new AdsFandomObject(driver, testedPage);
+    getJquery();
+
+    fandomPage.verifyTopLeaderboard();
+    fandomPage.verifyTopBoxad();
+    fandomPage.verifyNoBottomBoxad();
+    fandomPage.verifyNoBottomLeaderboard();
```

Comment:
I think sooner or later we need to start to use explicit `assertions` instead of `verify`.
E.g. 
`fandomPage.verifyNoBottomBoxad();`
becomes 
`assert.isNull(fandom.getSlot(fandom.BOTTOM_LEADERBOARD));`
(pseudocode)